src/fancy_dropdowns.js

```diff
@@ -5,14 +5,29 @@ $(document).ready(function() {
 
 	// Functions
 	var defaults = function(dest,source) {
-		for(var k in source) {
-			if(dest[k] === void 0) {
-				dest[k] = source[k];
-			}
-		}
+				for(var k in source) {
+					if(dest[k] === void 0) {
+						dest[k] = source[k];
+					}
+				}
 
-		return dest;
-	};
+				return dest;
+			},
+
+		  markupEscapeCodes = {
```

Comment:
@pdufour It looks to me like I'm using 4-space indents. Didn't realize that the rest of the file uses tabs, but can change this. I'm not exactly sure what you're expecting based on the image you posted, but will try to update as necessary. It might make sense to change the file to use 4-space indents as well since that is the widely accepted convention for JavaScript.